cookbooks/memcached/attributes/default.rb

```diff
@@ -15,15 +15,15 @@
   memcached['install_from_source'] = false
 
   # If you're installing from the portage tree, the latest available version is 1.4.25
-  memcached['version'] = '1.4.34'
-  memcached['download_url'] = 'https://memcached.org/files/memcached-1.4.34.tar.gz'
+  memcached['version'] = '1.4.39-r1'
+  #memcached['download_url'] = 'https://memcached.org/files/memcached-1.4.39-r1.tar.gz'
```

Comment:
Did you test this download url? I bet it returns a 404 (Memcached does not do `-r` releases, only we do in EYGL).

Reply:
Yes URL had issues. Changed the URL to work with 1.4.39